schunk_lwa4d_moveit_config/config/sensor_kinect.yaml

```diff
@@ -0,0 +1,7 @@
+sensors:
```

Comment:
well, there is no camera in our setup, but we could keep this file as an example...